resources/views/components/game/link-buttons/index.blade.php

```diff
@@ -97,9 +97,16 @@
     @endif
 
     @if ($canSeeOpenTickets)
+        @php
+            $href = route('game.tickets', ['game' => $game, 'filter[achievement]' => 'core']);
+            if (!$isViewingOfficial) {
+                $href = route('game.tickets', ['game' => $game, 'filter[achievement]' => 'unofficial']);
+            }
```

Comment:
Seems like this should be an if/else instead of calculating the core route, then possibly overwriting it.

However, the overhead is probably minimal, and the unofficial page isn't commonly used, so I'm not opposed to leaving it this way.

Reply:
Went ahead and adjusted this in https://github.com/RetroAchievements/RAWeb/pull/2605/commits/0385486465e3c912c5f7597d6f15f76de00245ef.